src/Nancy/DynamicDictionary.cs

```diff
@@ -3,11 +3,15 @@
     using System;
     using System.Collections;
     using System.Collections.Generic;
+    using System.Diagnostics;
     using System.Dynamic;
+    using System.Linq;
+    using System.Text;
 
     /// <summary>
     /// A dictionary that supports dynamic access.
     /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay}")]

```

Comment:
Change this to `[DebuggerDisplay("{DebuggerDisplay, nq}")]` (adding `, nq`, for "no quotes").